tools/test-all-ffb

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e
+
+help() {
+    echo -e "
+Test syntax using fake-flux-build helper. Must be run from repo root dir (a dir that contains
```

Comment:
If this scripts is always in this dir by convention and management-clusters dir should always be at the root of the repo, then we could do something like:

```bash
ROOT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )

# ...

for mc in $(find ${ROOT_DIR}/${mc_dir}/ ...
```